lib/unfurls/index.js

```diff
@@ -71,16 +78,17 @@ async function linkShared(req, res) {
           user: event.user,
           attachments: [new PrivateUnfurlPrompt(newUnfurl).getAttachment()],
         });
+      } else {
+        return Promise.resolve();
```

Comment:
Unless I'm missing something, I think this can just `return;` without the `Promise.resolve()`